tests/test_job.py

```diff
@@ -719,6 +719,15 @@ def test_create_job_with_id(self):
 
         self.assertRaises(TypeError, queue.enqueue, fixtures.say_hello, job_id=1234)
 
+    def test_create_job_with_async(self):
+        """test creating jobs with async function"""
+        queue = Queue(connection=self.testconn)
+        job = queue.enqueue(fixtures.say_hello_async, job_id="1234")
+        self.assertEqual(job.id, "1234")
+        job.perform()
```

Comment:
Thanks for this PR! Could you please add an assertion here to check that the result is right?

Reply:
Ready both comments